src/events/qaMention.js

```diff
@@ -34,17 +34,18 @@ module.exports = {
       const qaRoleIdRef = `<@&${qaRole.id}>`;
       const messageLink = `https://discord.com/channels/${message.guild.id}/${message.channel.id}/${message.id}`;
 
-      await qaRequestChannel.send(`
-        The role ${qaRoleIdRef} has been mentioned and a review has been requested. [Check message](${messageLink}).
-        > ${message.content}.\n\nPlease create a thread on this message to send your report.
-      `);
+      await qaRequestChannel.send(
+        translateLanguage('qaMention.qaRequest')
+          .replace('{{role}}', qaRoleIdRef)
+          .replace('{{link}}', messageLink)
+          .replace('{{content}}', message.content)
+      );
```

Comment:
translateLanguage function has a way of interpolating variables in the translate, just pass it as [second param](https://github.com/TeamNovaSoft/novabot/blob/main/src/languages/index.js#L63C33-L63C39)